### DISCUSSION TITLE
How to force gitversion to use SemVer 2.0 and not pad build number in NuGetVersionV2

### DISCUSSION THREAD

**Discussion by tig** [GitTools/GitVersion#3645] [Q&A] [answered]
What I get:

```ps
dotnet-gitversion /showvariable SemVer
1.13.6-pre.3
dotnet-gitversion /showvariable NuGetVersionV2
1.13.6-pre0003
```

What I want:
```ps
dotnet-gitversion /showvariable NuGetVersionV2
1.13.6-pre.3
```

I've searched high and low how to configure gitversion to change this behavior. Please help!

**Comment by HHobeck** [comment, marked as answer]
Input of @arturcic 

> Better they use V6. This variable if I remember correctly was removed on V6 and there is only one nuget variable

I think the NuGetVersion variables are gone in v6. I would suggest to use `SemVer` variable instead. Please see the following example to see the supported variables:

https://github.com/GitTools/GitVersion/blob/main/src/GitVersion.Core.Tests/VersionCalculation/Approved/VariableProviderTests.ProvidesVariablesInContinuousDeploymentModeForStable.approved.txt
